_includes/settlement-reconciliation.md

```diff
@@ -127,22 +127,22 @@ transaction list. Below is a sequence diagram detailing the interaction.
 
 ```mermaid
 sequenceDiagram
-activate Merchant
-activate PayEx.FrontEnd
-activate PayEx.BackOffice
-Merchant-->>PayEx.FrontEnd: Online payment transactions
-deactivate Merchant
-
-PayEx.FrontEnd-->>PayEx.BackOffice: Transaction data
-deactivate PayEx.FrontEnd
-deactivate PayEx.BackOffice
-
-activate Merchant
-
-PayEx.BackOffice->>Merchant: Balance Report (PDF-file)
-note left of Merchant: files are sent by e-mail or file transfer
-PayEx.BackOffice->>Merchant: Transaction list (XLSX-file)
-PayEx.BackOffice->>Merchant: Transaction list (XML-file)
+    activate Merchant
+    activate PayEx.FrontEnd
```

Comment:
1. `PayEx` should be replaced with `SwedbankPay`
2. I don't think we need to separate between `FrontEnd` and `BackOffice`.
3. When using programmatic names with dots, underscores, CamelCase, etc., in diagrams, please provide an alias with `participant SwedbankPay as Swedbank Pay` in the start of the diagram.

Reply:
Updated